src/locale/locales/de/messages.po

```diff
@@ -15,41 +15,23 @@ msgstr ""
 
 #: src/view/com/modals/VerifyEmail.tsx:142
 msgid "(no email)"
-msgstr ""
-
-#: src/view/shell/desktop/RightNav.tsx:168
```

Comment:
Just so I understand, these translations have been removed entirely?

Reply:
Yes, all the lines that start with ~ are strings that are no longer in the codebase. If it happens that we need the translated strings in the future, we can always get them back from the history. But for now, it serves no purpose to include them in the messages.po file so I thought it best to remove them.